### PR TITLE
[Mirror] Makes headpat wagging a pref

### PR DESCRIPTION
**Pull request by YakumoChen**
Port of Citadel-Station-13/Citadel-Station-13#13004

### About The Pull Request

prefs for headpat wagging
### Why It's Good For The Game

edgelords do not wag my immersion is broken
### Changelog

🆑
add: prefs for headpat wagging
/🆑

### CODE REVIEW

**Comment by Oscari-pog** [comment]
salt pr

**Comment by YakumoChen** [comment]
nah, I like headpat wagging, the PR body is copy pasted.

**Comment by NopemanMcHalt** [comment]
Feel free to edit out following lines, I closed them myself.

> closes #3534
> closes #3539

**Comment by ThatGuyFromThatPlace** [comment]
I just realized... This puts the preference in your general preferences menu, rather than per character... This means that you have to manually set it each time you change characters if you don't want it on a particular character... This is... Actually less preferable than the quirk because of that...

**Comment by YakumoChen** [comment]
You can also just enable and disable it at will including midround

**Comment by Gandalf2k15** [comment]
Mandatory tail wagging is required

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -353,12 +353,12 @@
...
-				for(var/T in many_tails)
-					if(S.mutant_bodyparts[T] && dna.features[T] != "None")
-						emote("wag")
-						break
+			if(!(client?.prefs.cit_toggles & NO_AUTO_WAG))
+				if(S?.can_wag_tail(src) && !dna.species.is_wagging_tail())
+					var/static/list/many_tails = list("tail_human", "tail_lizard", "mam_tail")
+					for(var/T in many_tails)
+						if(S.mutant_bodyparts[T] && dna.features[T] != "None")
+							emote("wag")
```
Comment (Azarak) [inline code/modules/mob/living/carbon/carbon_defense.dm:361]:
```suggestion
							emote("wag")
							break
```

**Comment by YakumoChen** [comment]
e review when

**Pull request closed without merging**